chord_metadata_service/restapi/tests/utils.py

```diff
@@ -13,3 +15,21 @@ def get_response(viewname, obj):
         data=json.dumps(obj),
         content_type='application/json'
     )
+
+
+def load_local_json(file_name):
+    """
+    Helper function that can be used to load json files
+    that are in the same directory as the caller.
+
+    Caller dir is obtained from the call stack.
+    Only a file_name value is required.
+    """
+
+    frame = inspect.stack()[1]
+    module = inspect.getmodule(frame[0])
+    caller_path = os.path.dirname(module.__file__)
```

Comment:
since this was factored out, maybe worth using `pathlib` instead of `os.path` since it's a bit cleaner for this type of thing / path join below